### PR TITLE
Add piping helpers for other specs

### PR DESCRIPTION
**Pull request by domenic**
See https://github.com/whatwg/fetch/pull/1199.

***
<a href="https://whatpr.org/streams/1113.html" title="Last updated on Apr 5, 2021, 9:04 PM UTC (9caffa3)">Preview</a> | <a href="https://whatpr.org/streams/1113/4b5b3cd...9caffa3.html" title="Last updated on Apr 5, 2021, 9:04 PM UTC (9caffa3)">Diff</a>

### CODE REVIEW

**Comment by ricea** [comment]
Woah, I totally missed this. Sorry.

**Review by ricea — APPROVED** [review]
lgtm with nits.

File: index.bs [hunk 1]
```diff
@@ -6563,6 +6550,59 @@ Streams</cite> is an example of this technique, with its `sender.createEncodedSt
 Despite such endpoint pairs obeying the `readable`/`writable` property contract, it never makes
 sense to pass them to {{ReadableStream/pipeThrough()}}.
 
+<h3 id="other-specs-piping">Piping</h3>
+
+<div algorithm="ReadableStream pipe to">
+ The result of a {{ReadableStream}} |readable| <dfn export for="ReadableStream" lt="pipe|pipe
+ to|piped to|piping to">piped to</dfn>  to a {{WritableStream}} |writable|, given an optional
+ boolean <dfn export for="ReadableStream/pipe to"><var>preventClose</var></dfn> (default false), an
```
[the remark this answers is not in the document]
Comment (domenic) [reply index.bs:6558]: Good catch. I should probably let people use either; it'll just make the source more verbose.

File: index.bs [hunk 2]
```diff
@@ -6563,6 +6550,59 @@ Streams</cite> is an example of this technique, with its `sender.createEncodedSt
...
+    |preventCancel|, |signalArg|).
+
+ <p class="note">If one doesn't care about the promise returned, referencing this concept can be a
+ bit awkward. The best we can suggest is "[=ReadableStream/pipe=] <var ignore>readable</var> to <var
+ ignore>writable</var>".</p>
+</div>
+
+<div algorithm="ReadableStream pipe through">
+ The result of a {{ReadableStream}} |readable| <dfn export for="ReadableStream" lt="pipe
+ through|piping through">piped through</dfn> a {{TransformStream}} |transform|, given an optional
```
[the remark this answers is not in the document]
Comment (domenic) [reply index.bs:6578]: The latter would require doing JavaScript-observable Get()s, which introduces the possibility of sync exceptions. I think we should leave it as just `TransformStream` for now unless some spec comes along that needs the generic concept.

[the remark this answers is not in the document]
Comment (ricea) [reply index.bs:6578]: That's reasonable.

**Comment by domenic** [comment]
Given that you found several good nits last round and I just got my eyes dilated so can barely see what I'm typing, I'd appreciate another once-over from you before merging :)

**Comment by ricea** [comment]
I took another look. Still lgtm!

**Pull request merged**